Sources/ExyteChat/ChatView/Giphy/GiphyAttributionMarkView.swift

```diff
@@ -0,0 +1,32 @@
+import SwiftUI
+
+class CustomTabBarController: UITabBarController {
+  var tabBarHeight: CGFloat {
+    tabBar.frame.size.height
+  }
+}
+
```

Comment:
one of the requirements for getting your app approved to use the giphySDK is that you show an attribution mark. This requirement is not very well specified on the giphy sdk docs, however including this small image overlay on the picker meets the requirement